FxSsh/Services/UserauthArgs.cs

```diff
@@ -4,7 +4,7 @@ namespace FxSsh.Services
 {
     public class UserauthArgs
     {
-        public UserauthArgs(string keyAlgorithm, string fingerprint, byte[] key)
+        public UserauthArgs(Session s, string username, string keyAlgorithm, string fingerprint, byte[] key)
```

Comment:
rename `s` as `session`
```suggestion
        public UserauthArgs(Session session, string username, string keyAlgorithm, string fingerprint, byte[] key)
```